### PR TITLE
Remove { DEFAULT | = } and default_expr in uninstall_pgtap.sql

### PR DESCRIPTION
**Pull request by BronzeDeer**
Previously the Makefile produced the DROP Function statements by
copying the function definitions verbatim, however DROP FUNCTION
does not recognize default values and will therefore error when
trying to drop a function that the install script created with
default values for some or all arguments

### CODE REVIEW

**Review by theory — APPROVED** [review]

**Comment by BronzeDeer** [comment]
@theory  I think you forgot to merge this PR after approval or is there anything else I need to do before merge?

**Comment by theory** [comment]
No, kinda thought @decibel would do it, but I can.

**Pull request merged**
